src/main/webapp/app/shared/virtual-scroll/virtual-scroll.style.css

```diff
@@ -0,0 +1,19 @@
+:host {
```

Comment:
Name of the file does not follow out standards

Reply:
fixed